x/axelarnet/types/params.go

```diff
@@ -30,10 +30,10 @@ func KeyTable() params.KeyTable {
 // DefaultParams creates the default genesis parameters
 func DefaultParams() Params {
 	return Params{
-		SupportedChains:    []string{"Bitcoin", "Ethereum"},
+		SupportedChains:    []string{"Ethereum"},
 		RouteTimeoutWindow: 100,
 		MinAmount:          sdktypes.NewInt(1000000),
-		TransactionFeeRate: sdktypes.NewDecWithPrec(25, 5), // 0.025%
+		TransactionFeeRate: sdktypes.NewDecWithPrec(5, 5), // 0.05%
```

Comment:
you mean?
```suggestion
		TransactionFeeRate: sdktypes.NewDecWithPrec(5, 4), // 0.05%
```

Reply:
Haha, yeah, just noticed that. Changed it to 0.1% since our fee rate is applied on the source side instead of destination rn